python/pyarrow/includes/libarrow_gpu.pxd

```diff
@@ -0,0 +1,105 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# distutils: language = c++
+
+from pyarrow.includes.libarrow cimport *
+
+cdef extern from "arrow/gpu/cuda_api.h" namespace "arrow::gpu" nogil:
+
+    cdef cppclass CCudaDeviceManager" arrow::gpu::CudaDeviceManager":
+        @staticmethod
+        CStatus GetInstance(CCudaDeviceManager ** manager)
+        CStatus GetContext(int gpu_number, shared_ptr[CCudaContext] * ctx)
+        CStatus CreateNewContext(int gpu_number,
+                                 shared_ptr[CCudaContext] * ctx)
+        CStatus AllocateHost(int64_t nbytes,
+                             shared_ptr[CCudaHostBuffer] * buffer)
+        CStatus FreeHost(void * data, int64_t nbytes)
```

Comment:
Style is non-conforming -- remove the space before these pointer `*`

Reply:
Done.